pydra/engine/audit.py

```diff
@@ -173,15 +174,21 @@ def audit_task(self, task):
         import subprocess as sp
 
         label = task.name
-        if hasattr(task.inputs, "executable"):
-            command = task.cmdline
-        # assume function task
-        else:
-            command = None
-        # if hasattr(task.inputs, "in_file"):
-        #     input_file = task.inputs.in_file
-        # else:
-        #     input_file = None
+        entity_label = type(label)
+
+        command = task.cmdline if hasattr(task.inputs, "executable") else None
+
+        attr_list = attr_fields(task.inputs)
+        for attrs in attr_list:
+            if attrs.type in [File, Directory]:
+                input_name = attrs.name
+                input_path = os.path.abspath(getattr(task.inputs, input_name))
+                file_hash = hash_file(input_path)
+
+            else:
```

Comment:
I believe this is not needed. For now, we said that we will not create any entity for inputs that are not files or directories